index.js

```diff
@@ -136,9 +137,14 @@ exports.astNodeVisitor = {
               lines[0] += ' @classdesc';
             }
             if (node.superClass) {
-              // Add class inheritance information because JSDoc does not honor
-              // the ES6 class's `extends` keyword
-              if (leadingComment.value.indexOf('@extends') === -1) {
+              const extendsIndex = lines.findIndex(line => line.includes('@extends'));
+              if (extendsIndex !== -1) {
+                // Remove the generic type from the extends tag, for example:
+                // "@extends {ParentClass<GenericType>}" -> "@extends {ParentClass}"
+                lines[extendsIndex] = lines[extendsIndex].replace(extendsWithGenericTypeRegEx, '@extends {$1}');
```

Comment:
Couldn't you just remove the line with `@extends` here, and always add the `@extends` derived from the class's `extends` keyword below?

Reply:
yes, good suggestion

Reply:
done